real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Real Estate Property'
+
+    name = fields.Char(required=True, string="Name")              
```

Comment:
```suggestion
    name = fields.Char(string="Name", required=True)
```
There shouldn't be extra trailing space and please make sure string is the first parameter since it's the first default parameter, though it's not a strict rule but better to follow a order
You can refer this for the parameter order defined for Fields 
https://github.com/odoo/odoo/blob/b2e564f07f6883111da4581bccfb0f45c7588063/odoo/fields.py#L148-L158